nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -283,11 +285,14 @@ private Handler loadInitialWars(final Set<Bundle> bundles) {
 
         // load the web ui app
         final WebAppContext webUiContext = loadWar(webUiWar, CONTEXT_PATH_NIFI, frameworkClassLoader);
-        webUiContext.getInitParams().put("oidc-supported", String.valueOf(props.isOidcEnabled()));
-        webUiContext.getInitParams().put("saml-supported", String.valueOf(props.isSamlEnabled()));
-        webUiContext.getInitParams().put("saml-single-logout-supported", String.valueOf(props.isSamlSingleLogoutEnabled()));
```

Comment:
Does it mean that SAML and OICD won't be supported anymore?

Reply:
SAML and OICD are still supported. Previously there were Servlet filters which consumed these `init params` which handled this routing. However, in an effort to further decouple front end and back end we've removed these Servlet Filters here [1]. These `init params` should have been removed in [1] but they were overlooked. Currently there is nothing consuming these.

[1] https://github.com/apache/nifi/pull/8843